com.aobuchow.themes.spectrum.feature/feature.properties

```diff
@@ -0,0 +1,4 @@
+featureName=Spectrum Theme
+featureURL=https://github.com/AObuchow/Eclipse-Modern-Dark-Theme
```

Comment:
This URL needs to be updated when the github repo is renamed.

Reply:
Hm. Does this mean you have to move your Git repo, too? Maybe there's a way to rename Githup repos, too... - or to move the issues from one repo to another, so you can keep all the tickets and their history.

Reply:
Yes there's a feature to rename repo's thankfully :D